tests/components/mikrotik/test_device_tracker.py

```diff
@@ -90,7 +90,7 @@ async def test_device_trackers(hass, mock_device_registry_devices):
         # test device_2 is added after connecting to wireless network
         WIRELESS_DATA.append(DEVICE_2_WIRELESS)
 
-        await hub.async_update()
+        await hub.async_refresh()
```

Comment:
We should not access integration details like hass.data or the coordinator in the tests. Please just set up the integration and move time forward as needed to make the coordinator refresh the data.

https://developers.home-assistant.io/docs/development_testing#writing-tests-for-integrations